Frontend/student-hub/src/components/Navbar.js

```diff
@@ -85,17 +85,14 @@ function Navbar({ isLoggedIn, setIsLoggedIn}) {
                             open={Boolean(settingsEl)}
                             onClose={handleSettingsClose}
                             >
-                                <MenuItem>
-                                <Switch checked={darkMode} onChange={toggleDarkMode} />
-                                Dark Mode
-                                </MenuItem>
                                 <MenuItem onClick={() => { navigate('/feedback'); handleSettingsClose(); }}>Feedback</MenuItem>
                                 <MenuItem onClick={() => { navigate('/contact'); handleSettingsClose(); }}>Contact Me</MenuItem>
                                 <MenuItem onClick={() => { navigate('/about'); handleSettingsClose(); }}>About</MenuItem>
                             </Menu>
                     </>
                 ) : (
                     <>
+
                     </>
```

Comment:
instead of an empty <></> you should use `undefined`

Reply:
I have made this change